Python build error: fatal error: Python.h: No such file or directory

This doesn't cause the build to actually fail, but makes the melange output noisy. maybe solved by adding the python-dev package

```
2022/07/14 17:33:32 melange (hello-server/armv7):   Preparing metadata (setup.py): still running...
2022/07/14 17:33:32 melange (hello-server/armv7):   Preparing metadata (setup.py): finished with status 'done'
2022/07/14 17:33:32 melange (hello-server/armv7): Building wheels for collected packages: MarkupSafe
2022/07/14 17:33:32 melange (hello-server/armv7):   Building wheel for MarkupSafe (setup.py): started
2022/07/14 17:34:36 melange (hello-server/armv7):   Building wheel for MarkupSafe (setup.py): still running...
2022/07/14 17:34:47 melange (hello-server/armv7):   Building wheel for MarkupSafe (setup.py): finished with status 'error'
2022/07/14 17:34:48 melange (hello-server/armv7):   error: subprocess-exited-with-error
2022/07/14 17:34:48 melange (hello-server/armv7):   
2022/07/14 17:34:48 melange (hello-server/armv7):   × python setup.py bdist_wheel did not run successfully.
2022/07/14 17:34:48 melange (hello-server/armv7):   │ exit code: 1
2022/07/14 17:34:48 melange (hello-server/armv7):   ╰─> [178 lines of output]
2022/07/14 17:34:48 melange (hello-server/armv7):       running bdist_wheel
2022/07/14 17:34:48 melange (hello-server/armv7):       running build
2022/07/14 17:34:48 melange (hello-server/armv7):       running build_py
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/lib.linux-armv7l-cpython-310
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/lib.linux-armv7l-cpython-310/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/_native.py -> build/lib.linux-armv7l-cpython-310/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/__init__.py -> build/lib.linux-armv7l-cpython-310/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       running egg_info
2022/07/14 17:34:48 melange (hello-server/armv7):       writing src/MarkupSafe.egg-info/PKG-INFO
2022/07/14 17:34:48 melange (hello-server/armv7):       writing dependency_links to src/MarkupSafe.egg-info/dependency_links.txt
2022/07/14 17:34:48 melange (hello-server/armv7):       writing top-level names to src/MarkupSafe.egg-info/top_level.txt
2022/07/14 17:34:48 melange (hello-server/armv7):       reading manifest file 'src/MarkupSafe.egg-info/SOURCES.txt'
2022/07/14 17:34:48 melange (hello-server/armv7):       reading manifest template 'MANIFEST.in'
2022/07/14 17:34:48 melange (hello-server/armv7):       no previously-included directories found matching 'docs/_build'
2022/07/14 17:34:48 melange (hello-server/armv7):       warning: no previously-included files matching '*.pyc' found anywhere in distribution
2022/07/14 17:34:48 melange (hello-server/armv7):       adding license file 'LICENSE.rst'
2022/07/14 17:34:48 melange (hello-server/armv7):       writing manifest file 'src/MarkupSafe.egg-info/SOURCES.txt'
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/_speedups.c -> build/lib.linux-armv7l-cpython-310/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/_speedups.pyi -> build/lib.linux-armv7l-cpython-310/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/py.typed -> build/lib.linux-armv7l-cpython-310/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       running build_ext
2022/07/14 17:34:48 melange (hello-server/armv7):       building 'markupsafe._speedups' extension
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/temp.linux-armv7l-cpython-310
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/temp.linux-armv7l-cpython-310/src
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/temp.linux-armv7l-cpython-310/src/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fomit-frame-pointer -g -fomit-frame-pointer -g -fomit-frame-pointer -g -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/include -I/usr/include/python3.10 -c src/markupsafe/_speedups.c -o build/temp.linux-armv7l-cpython-310/src/markupsafe/_speedups.o
2022/07/14 17:34:48 melange (hello-server/armv7):       src/markupsafe/_speedups.c:1:10: fatal error: Python.h: No such file or directory
2022/07/14 17:34:48 melange (hello-server/armv7):           1 | #include <Python.h>
2022/07/14 17:34:48 melange (hello-server/armv7):             |          ^~~~~~~~~~
2022/07/14 17:34:48 melange (hello-server/armv7):       compilation terminated.
2022/07/14 17:34:48 melange (hello-server/armv7):       ==========================================================================
2022/07/14 17:34:48 melange (hello-server/armv7):       WARNING: The C extension could not be compiled, speedups are not enabled.
2022/07/14 17:34:48 melange (hello-server/armv7):       Failure information, if any, is above.
2022/07/14 17:34:48 melange (hello-server/armv7):       Retrying the build without the C extension now.
2022/07/14 17:34:48 melange (hello-server/armv7):       ==========================================================================
2022/07/14 17:34:48 melange (hello-server/armv7):       running bdist_wheel
2022/07/14 17:34:48 melange (hello-server/armv7):       running build
2022/07/14 17:34:48 melange (hello-server/armv7):       running build_py
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/lib
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/lib/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/_native.py -> build/lib/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/__init__.py -> build/lib/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       running egg_info
2022/07/14 17:34:48 melange (hello-server/armv7):       writing src/MarkupSafe.egg-info/PKG-INFO
2022/07/14 17:34:48 melange (hello-server/armv7):       writing dependency_links to src/MarkupSafe.egg-info/dependency_links.txt
2022/07/14 17:34:48 melange (hello-server/armv7):       writing top-level names to src/MarkupSafe.egg-info/top_level.txt
2022/07/14 17:34:48 melange (hello-server/armv7):       reading manifest file 'src/MarkupSafe.egg-info/SOURCES.txt'
2022/07/14 17:34:48 melange (hello-server/armv7):       reading manifest template 'MANIFEST.in'
2022/07/14 17:34:48 melange (hello-server/armv7):       no previously-included directories found matching 'docs/_build'
2022/07/14 17:34:48 melange (hello-server/armv7):       warning: no previously-included files matching '*.pyc' found anywhere in distribution
2022/07/14 17:34:48 melange (hello-server/armv7):       adding license file 'LICENSE.rst'
2022/07/14 17:34:48 melange (hello-server/armv7):       writing manifest file 'src/MarkupSafe.egg-info/SOURCES.txt'
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/_speedups.c -> build/lib/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/_speedups.pyi -> build/lib/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying src/markupsafe/py.typed -> build/lib/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       /home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
2022/07/14 17:34:48 melange (hello-server/armv7):         warnings.warn(
2022/07/14 17:34:48 melange (hello-server/armv7):       installing to build/bdist.linux-armv7l/wheel
2022/07/14 17:34:48 melange (hello-server/armv7):       running install
2022/07/14 17:34:48 melange (hello-server/armv7):       running install_lib
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/bdist.linux-armv7l
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/bdist.linux-armv7l/wheel
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/bdist.linux-armv7l/wheel/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying build/lib/markupsafe/_native.py -> build/bdist.linux-armv7l/wheel/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying build/lib/markupsafe/_speedups.c -> build/bdist.linux-armv7l/wheel/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying build/lib/markupsafe/_speedups.pyi -> build/bdist.linux-armv7l/wheel/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying build/lib/markupsafe/__init__.py -> build/bdist.linux-armv7l/wheel/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       copying build/lib/markupsafe/py.typed -> build/bdist.linux-armv7l/wheel/markupsafe
2022/07/14 17:34:48 melange (hello-server/armv7):       running install_egg_info
2022/07/14 17:34:48 melange (hello-server/armv7):       Copying src/MarkupSafe.egg-info to build/bdist.linux-armv7l/wheel/MarkupSafe-2.1.1-py3.10.egg-info
2022/07/14 17:34:48 melange (hello-server/armv7):       running install_scripts
2022/07/14 17:34:48 melange (hello-server/armv7):       adding license file "LICENSE.rst" (matched pattern "LICENSE.rst")
2022/07/14 17:34:48 melange (hello-server/armv7):       creating build/bdist.linux-armv7l/wheel/MarkupSafe-2.1.1.dist-info/WHEEL
2022/07/14 17:34:48 melange (hello-server/armv7):       creating '/tmp/pip-wheel-c_1ujt3m/MarkupSafe-2.1.1-py3-none-any.whl' and adding 'build/bdist.linux-armv7l/wheel' to it
2022/07/14 17:34:48 melange (hello-server/armv7):       Traceback (most recent call last):
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/unixccompiler.py", line 173, in _compile
2022/07/14 17:34:48 melange (hello-server/armv7):           self.spawn(compiler_so + cc_args + [src, '-o', obj] +
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/ccompiler.py", line 917, in spawn
2022/07/14 17:34:48 melange (hello-server/armv7):           spawn(cmd, dry_run=self.dry_run, **kwargs)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/spawn.py", line 68, in spawn
2022/07/14 17:34:48 melange (hello-server/armv7):           raise DistutilsExecError(
2022/07/14 17:34:48 melange (hello-server/armv7):       distutils.errors.DistutilsExecError: command '/usr/bin/gcc' failed with exit code 1
2022/07/14 17:34:48 melange (hello-server/armv7):       
2022/07/14 17:34:48 melange (hello-server/armv7):       During handling of the above exception, another exception occurred:
2022/07/14 17:34:48 melange (hello-server/armv7):       
2022/07/14 17:34:48 melange (hello-server/armv7):       Traceback (most recent call last):
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/tmp/pip-install-rqehg8ix/markupsafe_8083186d4ec54b6f925891cb09818f77/setup.py", line 30, in build_extension
2022/07/14 17:34:48 melange (hello-server/armv7):           build_ext.build_extension(self, ext)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/command/build_ext.py", line 202, in build_extension
2022/07/14 17:34:48 melange (hello-server/armv7):           _build_ext.build_extension(self, ext)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 528, in build_extension
2022/07/14 17:34:48 melange (hello-server/armv7):           objects = self.compiler.compile(sources,
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/ccompiler.py", line 574, in compile
2022/07/14 17:34:48 melange (hello-server/armv7):           self._compile(obj, src, ext, cc_args, extra_postargs, pp_opts)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/unixccompiler.py", line 176, in _compile
2022/07/14 17:34:48 melange (hello-server/armv7):           raise CompileError(msg)
2022/07/14 17:34:48 melange (hello-server/armv7):       distutils.errors.CompileError: command '/usr/bin/gcc' failed with exit code 1
2022/07/14 17:34:48 melange (hello-server/armv7):       
2022/07/14 17:34:48 melange (hello-server/armv7):       The above exception was the direct cause of the following exception:
2022/07/14 17:34:48 melange (hello-server/armv7):       
2022/07/14 17:34:48 melange (hello-server/armv7):       Traceback (most recent call last):
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/tmp/pip-install-rqehg8ix/markupsafe_8083186d4ec54b6f925891cb09818f77/setup.py", line 65, in <module>
2022/07/14 17:34:48 melange (hello-server/armv7):           run_setup(True)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/tmp/pip-install-rqehg8ix/markupsafe_8083186d4ec54b6f925891cb09818f77/setup.py", line 41, in run_setup
2022/07/14 17:34:48 melange (hello-server/armv7):           setup(
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/__init__.py", line 87, in setup
2022/07/14 17:34:48 melange (hello-server/armv7):           return distutils.core.setup(**attrs)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 148, in setup
2022/07/14 17:34:48 melange (hello-server/armv7):           return run_commands(dist)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
2022/07/14 17:34:48 melange (hello-server/armv7):           dist.run_commands()
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
2022/07/14 17:34:48 melange (hello-server/armv7):           self.run_command(cmd)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/dist.py", line 1214, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           super().run_command(command)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           cmd_obj.run()
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/wheel/bdist_wheel.py", line 299, in run
2022/07/14 17:34:48 melange (hello-server/armv7):           self.run_command('build')
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           self.distribution.run_command(command)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/dist.py", line 1214, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           super().run_command(command)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           cmd_obj.run()
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/command/build.py", line 136, in run
2022/07/14 17:34:48 melange (hello-server/armv7):           self.run_command(cmd_name)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           self.distribution.run_command(command)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/dist.py", line 1214, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           super().run_command(command)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           cmd_obj.run()
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/tmp/pip-install-rqehg8ix/markupsafe_8083186d4ec54b6f925891cb09818f77/setup.py", line 24, in run
2022/07/14 17:34:48 melange (hello-server/armv7):           build_ext.run(self)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/command/build_ext.py", line 79, in run
2022/07/14 17:34:48 melange (hello-server/armv7):           _build_ext.run(self)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 339, in run
2022/07/14 17:34:48 melange (hello-server/armv7):           self.build_extensions()
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 448, in build_extensions
2022/07/14 17:34:48 melange (hello-server/armv7):           self._build_extensions_serial()
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 473, in _build_extensions_serial
2022/07/14 17:34:48 melange (hello-server/armv7):           self.build_extension(ext)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/tmp/pip-install-rqehg8ix/markupsafe_8083186d4ec54b6f925891cb09818f77/setup.py", line 32, in build_extension
2022/07/14 17:34:48 melange (hello-server/armv7):           raise BuildFailed() from e
2022/07/14 17:34:48 melange (hello-server/armv7):       __main__.BuildFailed
2022/07/14 17:34:48 melange (hello-server/armv7):       
2022/07/14 17:34:48 melange (hello-server/armv7):       During handling of the above exception, another exception occurred:
2022/07/14 17:34:48 melange (hello-server/armv7):       
2022/07/14 17:34:48 melange (hello-server/armv7):       Traceback (most recent call last):
2022/07/14 17:34:48 melange (hello-server/armv7):         File "<string>", line 2, in <module>
2022/07/14 17:34:48 melange (hello-server/armv7):         File "<pip-setuptools-caller>", line 34, in <module>
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/tmp/pip-install-rqehg8ix/markupsafe_8083186d4ec54b6f925891cb09818f77/setup.py", line 73, in <module>
2022/07/14 17:34:48 melange (hello-server/armv7):           run_setup(False)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/tmp/pip-install-rqehg8ix/markupsafe_8083186d4ec54b6f925891cb09818f77/setup.py", line 41, in run_setup
2022/07/14 17:34:48 melange (hello-server/armv7):           setup(
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/__init__.py", line 87, in setup
2022/07/14 17:34:48 melange (hello-server/armv7):           return distutils.core.setup(**attrs)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 148, in setup
2022/07/14 17:34:48 melange (hello-server/armv7):           return run_commands(dist)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
2022/07/14 17:34:48 melange (hello-server/armv7):           dist.run_commands()
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
2022/07/14 17:34:48 melange (hello-server/armv7):           self.run_command(cmd)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/dist.py", line 1214, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           super().run_command(command)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
2022/07/14 17:34:48 melange (hello-server/armv7):           cmd_obj.run()
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/wheel/bdist_wheel.py", line 361, in run
2022/07/14 17:34:48 melange (hello-server/armv7):           wf.write_files(archive_root)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/wheel/wheelfile.py", line 136, in write_files
2022/07/14 17:34:48 melange (hello-server/armv7):           self.write(path, arcname)
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/lib/python3.10/site-packages/wheel/wheelfile.py", line 147, in write
2022/07/14 17:34:48 melange (hello-server/armv7):           zinfo = ZipInfo(arcname or filename, date_time=get_zipinfo_datetime(st.st_mtime))
2022/07/14 17:34:48 melange (hello-server/armv7):         File "/usr/lib/python3.10/zipfile.py", line 362, in __init__
2022/07/14 17:34:48 melange (hello-server/armv7):           raise ValueError('ZIP does not support timestamps before 1980')
2022/07/14 17:34:48 melange (hello-server/armv7):       ValueError: ZIP does not support timestamps before 1980
2022/07/14 17:34:48 melange (hello-server/armv7):       [end of output]
2022/07/14 17:34:48 melange (hello-server/armv7):   
2022/07/14 17:34:48 melange (hello-server/armv7):   note: This error originates from a subprocess, and is likely not a problem with pip.
2022/07/14 17:34:48 melange (hello-server/armv7):   ERROR: Failed building wheel for MarkupSafe
2022/07/14 17:34:48 melange (hello-server/armv7):   Running setup.py clean for MarkupSafe
2022/07/14 17:34:59 melange (hello-server/armv7): Failed to build MarkupSafe
2022/07/14 17:35:00 melange (hello-server/armv7): Installing collected packages: Werkzeug, MarkupSafe, itsdangerous, click, Jinja2, Flask
2022/07/14 17:35:03 melange (hello-server/armv7):   Running setup.py install for MarkupSafe: started
2022/07/14 17:36:04 melange (hello-server/armv7):   Running setup.py install for MarkupSafe: still running...
2022/07/14 17:36:11 melange (hello-server/armv7):   Running setup.py install for MarkupSafe: finished with status 'done'
2022/07/14 17:36:11 melange (hello-server/armv7):   DEPRECATION: MarkupSafe was installed using the legacy 'setup.py install' method, because a wheel could not be built for it. A possible replacement is to fix the wheel build issue reported above. Discussion can be found at https://github.com/pypa/pip/issues/8368
2022/07/14 17:36:15 melange (hello-server/armv7): Successfully installed Flask-2.1.2 Jinja2-3.1.2 MarkupSafe-2.1.1 Werkzeug-2.1.2 click-8.1.3 itsdangerous-2.1.2
2022/07/14 17:36:17 melange (hello-server/armv7): WARNING: You are using pip version 22.0.4; however, version 22.1.2 is available.
2022/07/14 17:36:17 melange (hello-server/armv7): You should consider upgrading via the '/home/build/melange-out/hello-server/usr/share/webapps/hello-server/venv/bin/python -m pip install --upgrade pip' command.
2022/07/14 17:36:18 melange (hello-server/armv7): warning: read |0: file already closed
2022/07/14 17:36:18 melange (hello-server/armv7): warning: read |0: file already closed
2022/07/14 17:36:18 melange (hello-server/armv7): generating package hello-server-0.1.0-r0
2022/07/14 17:36:18 melange (hello-server/armv7): scanning for shared object dependencies...
2022/07/14 17:36:18 melange (hello-server/armv7): scanning for commands...
2022/07/14 17:36:18 melange (hello-server/armv7):   runtime:
2022/07/14 17:36:18 melange (hello-server/armv7):     python3
2022/07/14 17:36:18 melange (hello-server/armv7):   provides:
2022/07/14 17:36:18 melange (hello-server/armv7):     cmd:hello-server=0.1.0-r0
2022/07/14 17:36:18 melange (hello-server/armv7):   installed-size: 2[1435](https://github.com/chainguard-dev/hello-melange-apko/runs/7344995367?check_suite_focus=true#step:3:1472)686
2022/07/14 17:36:19 melange (hello-server/armv7):   data.tar.gz digest: a627616ff0eaad2c011e0fee1f42eeef7a526db63100309188f80bfcd3bd8133
2022/07/14 17:36:19 melange (hello-server/armv7):   control.tar.gz digest: 770a15256e36600233ac0a74c2638ca248b0e6e6
2022/07/14 17:36:19 melange (hello-server/armv7): wrote /home/runner/work/hello-melange-apko/hello-melange-apko/packages/armv7/hello-server-0.1.0-r0.apk
```

--- a/py/melange.yaml
+++ b/py/melange.yaml
@@ -24,6 +24,7 @@ environment:
       - gcc
       - musl-dev
       - python3
+      - python3-dev
       - py3-pip
       - py3-virtualenv
 

